spa/src/components/account/common/leftbar/Leftbar.styled.js

```diff
@@ -0,0 +1,121 @@
+import styled from 'styled-components';
+import SvgIcon from 'assets/icons/SvgIcon';
+
+export const Leftbar = styled.div`
+  width: 85%;
+  max-width: 506px;
+  margin-bottom: 50px;
+
+  color: ${(props) =>
+    props.bgColor && props.bgColor === 'purple' ? props.theme.colors.white : props.theme.colors.account.purple[1]};
```

Comment:
If you use the BG_COLOR variable here as well, probably need to create a constant file to have the declaration to avoid circular imports
```suggestion
    props.bgColor && props.bgColor === BG_COLOR.PURPLE ? props.theme.colors.white : props.theme.colors.account.purple[1]};
```

Reply:
done